lib/main.ts

```diff
@@ -135,14 +135,18 @@ export class Transpiler {
 
   private createCompilerHost(files: string[]): ts.CompilerHost {
     var defaultLibFileName = ts.getDefaultLibFileName(COMPILER_OPTIONS);
+    defaultLibFileName = this.normalizeSlashes(defaultLibFileName);
     return {
       getSourceFile: (sourceName, languageVersion) => {
         var path = sourceName;
         if (sourceName === defaultLibFileName) {
           path = ts.getDefaultLibFilePath(COMPILER_OPTIONS);
         }
+        // if (process.platform === 'win32') path = sourceName.replace(/\//g, '\\');
         if (!fs.existsSync(path)) return undefined;
         var contents = fs.readFileSync(path, 'UTF-8');
+        // Make sure the file path is relative and all file names use forward slashes.
+        var sourceName = this.getRelativeFileName(sourceName);
```

Comment:
Passing the relative path here generates some failures locally when running the Dart unit tests of Angular2. It seems that the file cache of TS is going wrong (something like picking the original source instead of a transformed file from a Broccoli tree):
```
[00:33:07] Error: [TSToDartTranspiler]: Debug Failure. False expression: end mus
t be the bounds of the file. 531 > 373
    at Object.assert (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:1379:23)
    at Object.createFileDiagnostic (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:999:15)
    at getSourceFileFromCache (c:\Dev\angular_mlaval\node_modules\ts2dart\node_m
odules\typescript\bin\typescript.js:29901:44)
    at findSourceFile (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:29861:24)
    at processSourceFile (c:\Dev\angular_mlaval\node_modules\ts2dart\node_module
s\typescript\bin\typescript.js:29831:27)
    at c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:29910:17
    at Object.forEach (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:718:30)
    at processReferencedFiles (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:29908:16)
    at findSourceFile (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\t
ypescript\bin\typescript.js:29884:25)
    at processSourceFile (c:\Dev\angular_mlaval\node_modules\ts2dart\node_modules\typescript\bin\typescript.js:29831:27)
```

Going back to `var sourceName = this.normalizeSlashes(sourceName);` solves this issue and allows to run the tests successfully :)